src/const.h

```diff
@@ -517,6 +517,27 @@ enum PlayerFlags : uint64_t {
 	PlayerFlag_IsAlwaysPremium = static_cast<uint64_t>(1) << 37,
 };
 
+enum ReloadTypes_t {
+	RELOAD_TYPE_ACTIONS = 1,
+	RELOAD_TYPE_CONFIG = 2,
+	RELOAD_TYPE_CREATURESCRIPTS = 3,
+	RELOAD_TYPE_MONSTERS = 4,
+	RELOAD_TYPE_MOVEMENTS = 5,
+	RELOAD_TYPE_NPCS = 6,
+	RELOAD_TYPE_RAIDS = 7,
+	RELOAD_TYPE_SPELLS = 8,
+	RELOAD_TYPE_TALKCTIONS = 9,
+	RELOAD_TYPE_ITEMS = 10,
+	RELOAD_TYPE_WEAPONS = 11,
+	RELOAD_TYPE_QUESTS = 12,
+	RELOAD_TYPE_MOUNTS = 13,
+	RELOAD_TYPE_GLOBALEVENTS = 14,
+	RELOAD_TYPE_EVENTS = 15,
+	RELOAD_TYPE_CHAT = 16,
+	RELOAD_TYPE_GLOBAL = 17,
+	RELOAD_TYPE_COMMANDS = 18, // Planned to be removed, so let's put it last
```

Comment:
You probably don't need to define values for each one, only for commands where you can `= <last enum value> + 1`. It would also be much better to have them alphabetically ordered.

Add a `RELOAD_TYPE_ALL` at the top so that one can reload everything when not using a parameter.